content/api-reference/queries/get-activities.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: Get activities
+description: Retrieve all or very specific activities in your tenant
+---
+
+## Request
+
+### Query
+
+```graphql
+
+```
```

Comment:
```suggestion
```graphql
query GetActivities(
    $activity_type: [String!],
    $activity_status: [String!],
    $pathway_definition_id: [String!],
    $action: [String!],
    $patient_id: String,
    $offset: Float,
    $count: Float,
    ) {
  activities(      
      filters: {
          activity_type: {
              in: $activity_type
          }
          action: {
              in: $action
          }
          pathway_definition_id: {
              in: $pathway_definition_id
          }
          patient_id: {
              eq: $patient_id
          }
          activity_status: {
              in: $activity_status
          }
      }
      pagination: {
          offset: $offset
          count: $count
      }
    ) {
    success
    activities {
      stream_id
      id
      status
      action
      date
      object {
        id
        type
        name
      }
    }
  }
}
```
```